adapters/repos/db/inverted/prop_length_tracker_test.go

```diff
@@ -73,7 +73,7 @@ func Test_PropertyLengthTracker(t *testing.T) {
 
```

Comment:
can you add the  "Test_PropertyLengthTracker_Overflow"-test for the new tracker?

Reply:
The new tracker can't overflow, so there's no need to test it.

It's one of the good features of the new tracker.